src/Compilers/VisualBasic/Portable/Lowering/StateMachineRewriter/StateMachineFieldSymbol.vb

```diff
@@ -55,7 +56,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                    slotIndex:=slotindex,
                    accessibility:=accessibility,
                    isReadOnly:=isReadOnly,
-                   isShared:=isShared)
+                   isShared:=isShared,
+                   isSpecialNameAndRuntimeSpecial:=isSpecialNameAndRuntimeSpecial)
```

Comment:
This change looks unintentional?

Reply:
@jmarolf This is intentional. otherwise `isSpecialNameAndRuntimeSpecial` isnot propagated not the constuctor chain